Index: app/src/main/java/com/example/hamza/karton/Fragments/CatagoryFragment.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/hamza/karton/Fragments/CatagoryFragment.java	(date 1493822845000)
+++ app/src/main/java/com/example/hamza/karton/Fragments/CatagoryFragment.java	(revision )
@@ -60,7 +60,7 @@
          pDialog.setMessage("please wait!!");
         if (!pDialog.isShowing())
             pDialog.show();
-        final String query = "PLWz5rJ2EKKc_Tt7q77qwyKRgytF1RzRx8";
+        final String query = "cartoon";
 
 
         try {
Index: app/src/main/java/com/example/hamza/karton/activities/MainActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/hamza/karton/activities/MainActivity.java	(date 1493822845000)
+++ app/src/main/java/com/example/hamza/karton/activities/MainActivity.java	(revision )
@@ -14,8 +14,8 @@
 import android.widget.Toast;
 
 import com.example.hamza.karton.Fragments.ActivityShareFragment;
+import com.example.hamza.karton.Fragments.CatagoryFragment;
 import com.example.hamza.karton.R;
-import com.example.hamza.karton.youtubeData.YouTubeActivity;
 
 public class MainActivity extends AppCompatActivity implements NavigationView.OnNavigationItemSelectedListener{
   // declaration here.........
@@ -54,7 +54,7 @@
         toggle.syncState();
 
         if (savedInstanceState == null) {
-            Fragment fragment = new YouTubeActivity();
+            Fragment fragment = new CatagoryFragment();
             navigationView.getMenu().getItem(0).setChecked(true);
             FragmentManager fragmentManager = getSupportFragmentManager();
             fragmentManager.beginTransaction().replace(R.id.flContent, fragment).commit();
@@ -78,7 +78,7 @@
         Fragment fragment = null;
         switch (id) {
             case R.id.Catagory:
-                fragment = new YouTubeActivity();
+                fragment = new CatagoryFragment();
                 break;
             case R.id.Search:
                 Toast.makeText(getApplication(),"search has selected..",Toast.LENGTH_SHORT).show();
