Index: app/src/main/java/com/example/hamza/karton/RecyclerView/AlbumsAdapter.java
===================================================================
--- app/src/main/java/com/example/hamza/karton/RecyclerView/AlbumsAdapter.java	(date 1493825253000)
+++ app/src/main/java/com/example/hamza/karton/adapters/AlbumsAdapter.java	
@@ -1,4 +1,4 @@
-package com.example.hamza.karton.RecyclerView;
+package com.example.hamza.karton.adapters;
 
 import android.app.Activity;
 import android.content.Context;
